container/features/src/main/resources/features.xml

```diff
@@ -1997,7 +2008,7 @@
         <feature>opennms-core-ipc-rpc-api</feature>
         <feature version="${opennmsApiVersion}">opennms-integration-api</feature>
         <feature>opennms-mate-impl</feature>
-        <feature>newts-api</feature>
+        <feature>opennms-newts-api</feature>
```

Comment:
Should we make a release for `newts` as 3.1 and use that here instead ?

https://github.com/OpenNMS/newts/blob/fddf2e28889914546f2f26aef43017a01be035ff/pom.xml#L58C6-L58C20

Looks like this already had an updated jackson version